client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js

```diff
@@ -296,10 +296,11 @@ class DetailedProgramSplit extends React.Component {
 
     // Increase height of the graph region for y-axis labels to have sufficient room
     // This is required to corretly display the labels when too many programs are present
-    const divHeight = Math.max(
-      650 * (graph_ready_data.length / 30) * 2,
-      80 // "magic" minimum height for the graph to render as desired
+    let divHeight = Math.max(
+      1000 * (graph_ready_data.length / 30) * 2,
+      100 // "magic" minimum height for the graph to render as desired
     );
+    divHeight = Math.min(divHeight, 1200); // max height of 1200 to prevent large graphs from becoming too large
```

Comment:
and `const divHeight = Math.min(minDivHeight...)` is clearer
Or you could do something like
```
_.chain([...])
        .max()
        .thru((max_val) => [max_val, 1200])
        .min()
        .value()
```